modules/database/src/ioc/db/dbIocRegister.c

```diff
@@ -46,7 +50,14 @@ static void dbLoadDatabaseCallFunc(const iocshArgBuf *args)
 static const iocshArg dbLoadRecordsArg0 = { "file name",iocshArgString};
 static const iocshArg dbLoadRecordsArg1 = { "substitutions",iocshArgString};
 static const iocshArg * const dbLoadRecordsArgs[2] = {&dbLoadRecordsArg0,&dbLoadRecordsArg1};
-static const iocshFuncDef dbLoadRecordsFuncDef = {"dbLoadRecords",2,dbLoadRecordsArgs};
+static const iocshFuncDef dbLoadRecordsFuncDef = {
+    "dbLoadRecords",
+    2,
+    dbLoadRecordsArgs,
+    "Load the given .db file, with the given substitutions.\n\n"
+    "Substitutions should be given in the format 'var1=value1,var2=value2'.\n"
+    "For more information about substitutions, see the 'macro substitution' documentation.\n",
```

Comment:
Please refer to the AppDevGuide for macro substitution docs.

Reply:
To my understanding, the AppDevGuide is to be deprecated in favor of a common ReadTheDocs documentation? I tried to be generic to prevent this documentation to be outdated.

Reply:
We have been moving away from and stopped updating the AppDevGuide several years ago (3.16.2 was the last version). There is a [Read-the-Docs version](https://docs.epics-controls.org/en/latest/appdevguide/AppDevGuide.html) of it, but the developers haven't been updating that either. I've been trying to find another reference for variable substitutions but can't fine anything obvious, other than the internals of macLib which does the substitution work. I would probably drop the last line since we can't point to anywhere obvious.